components/IndexFinalWinners.tsx

```diff
@@ -1,17 +1,43 @@
-import { Box, Container } from '@chakra-ui/react'
+import { Flex, Box, Container, Text } from '@chakra-ui/react'
 
 import AppSectionTitle from '@/components/AppSectionTitle'
 import styles from '@/styles/FinalWinners.module.scss'
 
-const IndexAwards = () => {
+export type Winner = {
+  title: string
+  team: string
+}
+
+type Props = {
+  winners: Winner[]
+}
+
+const IndexAwards = (props: Props) => {
   return (
-    <Container maxW="container.xl" pt={8} pb={2}>
+    <Container maxW="container.xl" pt={10} pb={10}>
       <Box as={'section'} className={styles.FinalWinners}>
         <AppSectionTitle
           enTitle="Final Review Result!"
-          jaTitle="受賞者の皆さま、おめでとうございました！！"
+          jaTitle="受賞者一覧"
+          textAlign="center"
         />
-        <Box pt={4}></Box>
+        <Flex wrap="wrap" justify="center">
+          {props.winners.map((winner, index) => (
+            <Box
+              flexBasis="33.3%"
+              key={index}
+              p={5}
+              textAlign="center"
+              minW="250px"
+              flex
+            >
+              <Text fontSize="1xl" whiteSpace="nowrap" margin="0 auto">
```

Comment:
```suggestion
              <Text fontSize="xl" whiteSpace="nowrap" margin="0 auto">
```
`1xl` というキーワードは無さそうです